_inc/client/components/navigation-settings/index.jsx

```diff
@@ -14,7 +14,7 @@ import { __, _x } from '@wordpress/i18n';
  */
 import analytics from 'lib/analytics';
 import { filterSearch, getSearchTerm } from 'state/search';
-import { isSiteConnected, isCurrentUserLinked } from 'state/connection';
```

Comment:
This can be cleaned regardless. Doesn't look like it's used anywhere in this component.